docs/docs/about/references/keywords/CABLE_LOSS.md

```diff
@@ -0,0 +1,28 @@
+# CABLE_LOSS
+
+[INSTALLATIONS](/about/references/keywords/INSTALLATIONS.md) /
+[GENERATORSETS](/about/references/keywords/GENERATORSETS.md)
+
+## Description
+
+Fraction, describing power loss from shore. Used to calculate actual power supply onshore. Relevant only for generatorsets of [CATEGORY](/about/references/keywords/CATEGORY.md) `POWER-FROM-SHORE`.
+
+## Format
+
+~~~~~~~~yaml
+
+CABLE_LOSS: <file or value>
```

Comment:
expression or value is what I would expect. Are we using "file" to describe expressions in docs?